public/src/managers/GRPCManager.ts

```diff
@@ -287,4 +290,10 @@ export class GrpcManager {
         });
         return grpcPromise;
     }
+    // if gRPC call returns with error, inform user (currently just logs to console)
+    private informUser(response: IGrpcResponse<any>, sender: string): void {
```

Comment:
I wonder if by use of reflection, you could find the calling method (`sender`) and thereby avoiding to pass the `sender` parameter. It would make it easier to use...

Reply:
Done.

Reply:
Nice